packages/blockchain-info-components/src/Images/Images.ts

```diff
@@ -269,6 +278,7 @@ const Images = {
   'us-bank': usBank,
   'us-flag': usFlag,
   'usdd-intro': usddIntro,
+  venmo,
   'visa-logo': visaLogo,
   'visa-new-logo': visaNewLogo,
   'wallet-v3-backup': walletV3Backup,
```

Comment:
we should have prefixed bank logos e.g.
```
bank-acorns: ..
bank-venmo: ..
```

Reply:
can we go back and clean up all the bank icons please?

Reply:
All the bank icons are in a sub folder `Images/img/BankIcons/` is that ok or should they still be named with `bank-logo-`?

Reply:
would be nice to have the icons labelled in case we need to use them in other places.  Who knows maybe some day we want an icon named "chase" that is a person running and not the bank. just better to specific IMO 🤷

Reply:
looks good!